### PR TITLE
[IOS-311]: Ping bots on conversation start

### PR DESCRIPTION
**Pull request by yuliaveres**
Did reload was not called if there were no messages on chatViewModel.
now we call it and initial message to bot is being sent.
also changed network alert text in this pr. !

### CODE REVIEW

File: Toshi/Supporting Files/AppDelegate.m [hunk 1]
```diff
@@ -120,7 +120,7 @@ - (void)configureAndPresentWindow {
 - (void)showNetworkAlertIfNeeded {
     // To drive this point really home we could show this for every launch instead.
     if (![[NSUserDefaults standardUserDefaults] boolForKey:@"DidShowMoneyAlert"]) {
-        UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"Be aware!" message:@"Toshi is running on Testnet. Do not send Ethereum from Mainnet." preferredStyle:UIAlertControllerStyleAlert];
+        UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"Be aware!" message:@"Toshi is not an appropriate wallet for storage of large sums of ETH." preferredStyle:UIAlertControllerStyleAlert];
```
Comment (roberthein) [inline Toshi/Supporting Files/AppDelegate.m]: Can you localize these strings?

File: Toshi/Controllers/Messaging/ChatController.swift [hunk 2]
```diff
@@ -231,6 +231,8 @@ final class ChatController: OverlayController {
 
         isVisible = true
 
+        viewModel.load()
```
Comment (roberthein) [inline Toshi/Controllers/Messaging/ChatController.swift]: The viewModel is not loading here, what is loading here? Maybe give `load()` more context?

Comment (roberthein) [reply Toshi/Controllers/Messaging/ChatController.swift]: Maybe something like `LoadFirstMessages`? What do you think?

File: Toshi/Supporting Files/AppDelegate.m (hunk 1 above)
Comment (yuliaveres) [reply Toshi/Supporting Files/AppDelegate.m]: yup

File: Toshi/Controllers/Messaging/ChatController.swift (hunk 2 above)
Comment (yuliaveres) [reply Toshi/Controllers/Messaging/ChatController.swift]: yup

**Pull request merged**